rero/rero-ils
# Admin interface: menu refactoring

* NEW: move sidebar to header
* BETTER: implement mobile first approach

Co-authored-by: Nicolas Labat <nicolas.labat@rero.ch>
Co-authored-by: Johnny Mariéthoz <johnny.mariéthoz@rero.ch>
Signed-off-by: Igor Milhit <igor.milhit@rero.ch>

-----------------
## To Test
- get the PR
- `./scripts/bootstrap`
- log in as a librarian
- load an admin page (circulation settings or my library)
- try the menu, especially ressources, my library, menu and my account (the user initals)
- check with large screen, medium screen, small screen

## Still to be done:
- bundles
- create menus structure as in the mockup, with the corrects icons

Should be done during and after refactoring the main layout (to bootstrap 4).
